ipapython/ipautil.py

```diff
@@ -127,12 +127,34 @@ def __init__(self, addr):
         super(UnsafeIPAddress, self).__init__(addr,
                                               flags=self.netaddr_ip_flags)
 
+    def __getstate__(self):
+        state = {attr: getattr(self, attr) for attr in ['_net']}
+        try:
+            state['super_state'] = super(UnsafeIPAddress, self).__getstate__()
+        except AttributeError:
+            # none of base classes implements custom pickling
+            pass
```

Comment:
We know `IPAddress` implements custom pickling, so the try block is unnecessary. Actually, if custom pickling was removed from `IPAddress`, the try block would break pickling completely, so it's not unnecessary, but harmful. Please remove it.